src/uct/ib/rdmacm/rdmacm_cm.c

```diff
@@ -281,23 +282,19 @@ uct_rdmacm_cm_reserved_qpn_blk_add(struct ibv_context *verbs,
     return status;
 }
 
-ucs_status_t
-uct_rdmacm_cm_reserved_qpn_blk_destroy(struct uct_rdmacm_cm_reserved_qpn_blk *blk)
+void uct_rdmacm_cm_reserved_qpn_blk_destroy(uct_rdmacm_cm_reserved_qpn_blk_t *blk)
 {
 #if HAVE_DECL_MLX5DV_IS_SUPPORTED
     ucs_assert(blk->refcount == 0);
 
     if (mlx5dv_devx_obj_destroy(blk->obj)) {
         ucs_error("mlx5dv_devx_obj_destroy(DESTROY_GENERAL_OBJECT, type=RESERVED_QPN) failed: %m");
-        return UCS_ERR_IO_ERROR;
+        return;
```

Comment:
can remove this 'return' and go-ahead to destroy blk even if devx obj destroy failed